chapters/03-package-setup.Rmd

```diff
@@ -16,4 +16,26 @@
 
 <!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
 
+1. Use `create_package()` to create a package called FIXME.  
 
+2. Edit the `DESCRIPTION` file to add:
+
+    * yourself as the author, and
+    * a one sentence description of the package.
+
+3. Add a LICENSE with `use_mit_license()`.
+
```

Comment:
I wasn't sure if this was premature - but it will be required to pass `check()` cleanly.

Reply:
Hmmm, that's a really good point... Let's keep this in mind and keep as is for now. Licensing makes most sense in the community involvement step, but we could preemptively get them to include it and say we will talk about it later.

Reply:
Right, given how early we are in the book (and the change from the next comment) let's delay it.